code/stable/glassbr/src/swift/InputParameters.swift

```diff
@@ -736,6 +736,26 @@ class InputParameters {
             print(".")
             throw "InputError"
         }
+        let set: Set<Double> = ["AN", "FT", "HS"]
```

Comment:
Oops... `Set<Double>` definitely won't work here!

Reply:
This was just changed in my last commit